the-badass-todo-list-part-1/list.js

```diff
@@ -29,11 +30,14 @@
     document.querySelector('#todo-list-wrapper').appendChild(list);
   }
 
+  // Quiz #2
   function saveData(todoItem, afterSave) {
```

Comment:
You got it. But that's the point why closure, especially if you put it as the only pattern people should follow strictly, is not a good solution, because of all the duplicated code.